mmv1/templates/terraform/post_create/compute_backend_bucket_security_policy.go.erb

```diff
@@ -0,0 +1,19 @@
+// security_policy isn't set by Create / Update
```

Comment:
This exists in the post_create, but what if the user goes from this field being unset on a bucket to setting it? Should the field be ForceNew or do we also need this in the Update call?

Reply:
Oh this is used as post_update as well, makes sense

Reply:
Can the update part of this be done by setting `update_url` on this specific field in the terraform.yaml? It allows calling a specific endpoint for a specific field